### ISSUE TITLE
Package 'dalmatian' has been archived on CRAN for more than 60 days

### ISSUE THREAD

**Issue by statmath-it** [cran-task-views/MixedModels#47]
Package [dalmatian](https://CRAN.R-project.org/package=dalmatian) is currently listed in CRAN Task View [MixedModels](https://CRAN.R-project.org/view=MixedModels) but the package has actually been archived for more than 60 days on CRAN.  Often this indicates that the package is currently not sufficiently actively maintained and should be excluded from the task view.

Alternatively, you might also consider reaching out to the authors of the package and encourage (or even help) them to bring the package back to CRAN.

In any case, the situation should be resolved in the next four weeks. If the package does not seem to be brought back to CRAN, please exclude it from the task view.

**Comment by palday** [comment]
Associated JSS Paper: https://www.jstatsoft.org/article/view/v100i10


Looks like the vignette builds are [failing](https://cran-archive.r-project.org/web/checks/2023/2023-04-20_check_results_dalmatian.html):
```
Version: 1.0.0
Check: re-building of vignette outputs
Result: ERROR
    Error(s) in re-building vignettes:
     ...
    --- re-building ‘beta-binomial-1.Rmd’ using rmarkdown
    Quitting from lines 142-147 (beta-binomial-1.Rmd)
    Error: processing vignette ‘beta-binomial-1.Rmd’ failed with diagnostics:
    could not find function "discrete_range"
    --- failed re-building ‘beta-binomial-1.Rmd’
    
    --- re-building ‘gamma-1.Rmd’ using rmarkdown
    Quitting from lines 144-149 (gamma-1.Rmd)
    Error: processing vignette ‘gamma-1.Rmd’ failed with diagnostics:
    could not find function "discrete_range"
    --- failed re-building ‘gamma-1.Rmd’
    
    --- re-building ‘negative-binomial-1.Rmd’ using rmarkdown
    Quitting from lines 144-149 (negative-binomial-1.Rmd)
    Error: processing vignette ‘negative-binomial-1.Rmd’ failed with diagnostics:
    could not find function "discrete_range"
    --- failed re-building ‘negative-binomial-1.Rmd’
    
    --- re-building ‘pied-flycatchers-1.Rmd’ using rmarkdown
    Quitting from lines 139-145 (pied-flycatchers-1.Rmd)
    Error: processing vignette ‘pied-flycatchers-1.Rmd’ failed with diagnostics:
    could not find function "discrete_range"
    --- failed re-building ‘pied-flycatchers-1.Rmd’
    
    --- re-building ‘pied-flycatchers-2.Rmd’ using rmarkdown
    Quitting from lines 124-136 (pied-flycatchers-2.Rmd)
    Error: processing vignette ‘pied-flycatchers-2.Rmd’ failed with diagnostics:
    could not find function "discrete_range"
    --- failed re-building ‘pied-flycatchers-2.Rmd’
    
    --- re-building ‘pied-flycatchers-3.Rmd’ using rmarkdown
    Quitting from lines 122-128 (pied-flycatchers-3.Rmd)
    Error: processing vignette ‘pied-flycatchers-3.Rmd’ failed with diagnostics:
    could not find function "discrete_range"
    --- failed re-building ‘pied-flycatchers-3.Rmd’
    
    --- re-building ‘weights-1-simulate.Rmd’ using rmarkdown
    --- finished re-building ‘weights-1-simulate.Rmd’
    
    SUMMARY: processing the following files failed:
     ‘beta-binomial-1.Rmd’ ‘gamma-1.Rmd’ ‘negative-binomial-1.Rmd’
     ‘pied-flycatchers-1.Rmd’ ‘pied-flycatchers-2.Rmd’
     ‘pied-flycatchers-3.Rmd’
    
    Error: Vignette re-building failed.
    Execution halted
```

**Comment by bbolker** [comment]
This isn't too hard to fix: https://github.com/sjbonner/dalmatian/pull/62 ; it's driven (I think) by incompatibility of some `ggplot` objects that were pre-computed and stored for use in the vignette with later versions of the `ggplot2` package ... (I also contacted the maintainer directly)

**Comment by jpiaskowski** [comment]
Are there any updates on this?

**Comment by bbolker** [comment]
resolved by @jpiaskowski  in 2813961
